hyperwallet/utils/apiclient.py

```diff
@@ -220,3 +222,25 @@ def __getRequestData(self, data):
         '''
 
         return (data if data is None else self.encryption.encrypt(data)) if self.encrypted else data
+
+    def putDocument(self, partialUrl, data, files):
```

Comment:
Please verify if the method has been added by you or if there is any duplication.

Reply:
v3 merged code